app/gui/src/controller/ide/desktop.rs

```diff
@@ -48,44 +48,49 @@ pub struct Handle {
 }
 
 impl Handle {
-    /// Create a project controller handle with already loaded project model.
-    pub fn new_with_project(
+    /// Create IDE controller. If `maybe_project_name` is `Some`, a project with provided name will
+    /// be opened. Otherwise controller will be used for project manager operations by Welcome
+    /// Screen.
+    pub async fn new(
```

Comment:
Is there a reason this needs to be async now?

Reply:
@vitvakatu please not mark comments as resolved if you were not the original author of a comment. It is the role of the person who was asking questions to mark a comment as resolved. This should be left open until @MichaelMauderer closes it.